tests/lib/util/glob-util.js

```diff
@@ -254,12 +245,13 @@ describe("globUtil", () => {
         it("should silently ignore default ignored files if not passed explicitly even if ignore is false", () => {
```

Comment:
Is this test name still accurate given that we have an `assert.throws` in the test body?